vcbuild.bat

```diff
@@ -242,12 +242,11 @@ if defined noprojgen if defined nobuild goto :after-build
 set msvs_host_arch=x86
 if _%PROCESSOR_ARCHITECTURE%_==_AMD64_ set msvs_host_arch=amd64
 if _%PROCESSOR_ARCHITEW6432%_==_AMD64_ set msvs_host_arch=amd64
+if _%PROCESSOR_ARCHITECTURE%_==_ARM64_ set msvs_host_arch=arm64
 @rem usually vcvarsall takes an argument: host + '_' + target
 set vcvarsall_arg=%msvs_host_arch%_%target_arch%
-@rem unless both host and target are x64
-if %target_arch%==x64 if %msvs_host_arch%==amd64 set vcvarsall_arg=amd64
-@rem also if both are x86
-if %target_arch%==x86 if %msvs_host_arch%==x86 set vcvarsall_arg=x86
+@rem unless both the host and the target are the same
+if %target_arch%==%msvs_host_arch% set vcvarsall_arg=%target_arch%
```

Comment:
This fails on Windows x64, since it compares "x64" to "amd64"

Reply:
OK, thank you. Let me fix that.

Reply:
Should be fixed now.

Reply:
Sorry for the delay, I was OOF last week. Yes, this is fixed now.